README.md

```diff
@@ -14,19 +14,43 @@ This repository is solely maintained by Docker, Inc.
 
 From `https://get.docker.com`:
 ```shell
-curl -fsSL get.docker.com -o get-docker.sh
-sh get-docker.sh
+$ curl -fsSL get.docker.com -o get-docker.sh
+$ sh get-docker.sh
 ```
 
 From `https://test.docker.com`:
 ```shell
-curl -fsSL test.docker.com -o test-docker.sh
-sh test-docker.sh
+$ curl -fsSL test.docker.com -o test-docker.sh
+$ sh test-docker.sh
 ```
 
 From the source repo (This will install latest from the `test` channel):
 ```shell
-sh install.sh
+$ sh install.sh
+```
+
+### Dry running
+
+If you'd like to see the commands that will be run for you distribution,
```

Comment:
I'd remove some of the "if you like" and "feel free to" wording. Perhaps something like:

```markdown
Before performing the actual installation, you can verify which commands
the script will run for your distribution. Use the `--dry run` option when
executing the script to print the commands without executing them:
```